File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.py
while(1):
    arr = input()[:5]
    prime = True
    if(arr == "0"): break
    for i in range(len(arr) // 2):
        if(arr[i] != arr[len(arr) - 1 - i]):
            prime = False
            break
    if(prime): print("yes")
    else: print("no")

File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.c
#include <stdio.h>
#include <string.h>


int main(void) {
	char arr[6];
	while (1) {
		scanf("%s", arr);
		if (strcmp(arr, "0") == 0) break;
		int palin = 1;
		for (int i = 0; i < strlen(arr); i++) {
			if (arr[i] != arr[strlen(arr) - i - 1]) {
				palin = 0;
				break;
			}
		}
		if (palin) printf("yes\n");
		else printf("no\n");

	}

	return 0;
}